internal/ethapi/api.go

```diff
@@ -1862,6 +1957,24 @@ func (api *PublicRollupAPI) GetInfo(ctx context.Context) rollupInfo {
 	}
 }
 
+// PrivatelRollupAPI provides private RPC methods to control the sequencer.
+// These methods can be abused by external users and must be considered insecure for use by untrusted users.
```

Comment:
Could this be replaced by periodically polling the data transport layer and updating the gas price? Happy to not make that a requirement of this PR, but it will make the devops a bit more challenging. Imagining that this should only be made available via IPC. I'm not aware of a way to make this API not available to the internet without a proxy in front of geth or making changes to how the geth RPC package works

Reply:
https://github.com/ethereum-optimism/data-transport-layer/pull/85

Reply:
I disagree that this is the best approach, you are tightly coupling the Data Service with Geth then. If the Data Service goes down, geth performance degrades. IMHO data should be pushed _TO_ geth over authenticated APIs, instead of expecting that Geth itself can pull data _FROM_ other services, which may break down individually.

Reply:
https://github.com/ethereum-optimism/go-ethereum/pull/277